blog/_posts/2020-07-20-three-architectures-for-responsive-ide.adoc

```diff
@@ -72,27 +72,27 @@ public class Main {
 
 The user has just typed `Foo.f().`, and we need to figure out that the type of receiver expression is `Bar`, and suggest `g` as a completion.
 
-First, as the `Main.java` file is modified, we run the indexer on this single file.
-Nothing has changed (the file still contains `Main` class with static `main` method), so we don't need to update FQN index.
+First, as the file `Main.java` is modified, we run the indexer on this single file.
```

Comment:
The word swap here and in a few other places might not be required for grammatical correctness (I'm not actually sure), but it makes the article easier to read IMHO.